src/main/java/org/geppetto/simulation/manager/GeppettoManager.java

```diff
@@ -78,6 +78,8 @@
 
 	private IGeppettoManagerCallbackListener geppettoManagerCallbackListener;
 
+	private boolean allowVolatileProjectsSimulation = false;
```

Comment:
This becomes GeppettoManagerConfiguration = new GeppettoManagerConfiguration(). This will be overridden with whatever is passed. The GeppettoManagerConfiguration by default will hold the default values.

Reply:
went away